src/common/components/FooterContent.tsx

```diff
@@ -1,14 +1,14 @@
 import { FC } from 'react';
 import { Constants } from 'utils/constants';
 
-const { version, creationYear } = Constants;
+const { version, creationYear, footerCopyright, environmentName } = Constants;
 const currentYear = new Date().getFullYear();
 
 export const copyrightDate: string =
   currentYear > creationYear ? `${creationYear} - ${currentYear}` : (`${creationYear}` as string);
 
 export const FooterContent: FC = () => (
   <span>
-    {copyrightDate} &copy; Bitwise Technology Consulting - {version} Staging
+    {copyrightDate} {footerCopyright} - {version} {environmentName}
```

Comment:
Refactored footer to make it data-driven and translatable, although in this case I'm not translating the environment name since it is a debugging tool.